renv/.gitignore

```diff
@@ -1,7 +1,7 @@
-sandbox/
+library/
```

Comment:
`renv/.gitignore`, `renv/activate.R`, and `renv/settings.dcf` all got automatically updated as part of the upgrade to renv 1.0.x and R 4.4.x.